### PR TITLE
Request/response benchmarks

### PR DESCRIPTION
**Pull request by pipermerriam**
unblocks https://github.com/ethereum/lahja/pull/98

## What was wrong?

Our benchmark tests do not cover the `EndpointAPI.request` code path.

## How was it fixed?

Added ability to target that API to benchmark suite using `--mode request`

#### Cute Animal Picture

cute-baby-hippos-7-590842bb42d16__700

### CODE REVIEW

File: lahja/tools/benchmark/process.py [hunk 1]
```diff
@@ -45,30 +49,56 @@ def stop(self) -> None:
...
 
-    @staticmethod
-    def launch(config: DriverProcessConfig) -> None:
+    @classmethod
+    def launch(cls, config: DriverProcessConfig) -> None:
         # UNCOMMENT FOR DEBUGGING
         # logger = multiprocessing.log_to_stderr()
         # logger.setLevel(logging.INFO)
-        config.backend.run(DriverProcess.worker, config)
+        config.backend.run(cls.worker, config)
```
Comment (lithp) [inline lahja/tools/benchmark/process.py]: Not part of this PR but: now that this method has been slimmed down so much `start()` could directly call `worker()`

**Comment by pipermerriam** [comment]
:+1: to a more extensive round trip benchmark for request/response.  opening an issue to track future work.
